mne/time_frequency/spectrum.py

```diff
@@ -1179,6 +1179,56 @@ def __getitem__(self, item):
         return BaseRaw._getitem(self, item, return_times=False)
 
 
+@fill_doc
+class SpectrumArray(Spectrum):
+    """Spectrum object from numpy array.
+
+    Parameters
+    ----------
+    data : array, shape (n_channels, n_freqs)
+        The channels' power spectral density for each epoch.
+    %(info_not_none)s
+    %(freqs_tfr)s
+    %(method_psd)s
+    %(verbose)s
+
+    See Also
+    --------
+    mne.create_info
+    mne.EpochsArray
+    mne.EvokedArray
+    mne.io.RawArray
+    EpochsSpectrumArray
```

Comment:
```suggestion
    mne.EvokedArray
    mne.io.RawArray
    EpochsSpectrumArray
    Spectrum
```
logic here is to only crossref classes with one degree of difference (i.e., EpochsArray is not linked because it is 2 degrees different from SpectrumArray: epoched & not spectral)